plugins/bitwarden/help.md

```diff
@@ -126,229 +112,168 @@ Example input:
 ##### Output
 
 |Name|Type|Required|Description|Example|
-|----|----|--------|-----------|-------|
-|member|member|False|An organization member|{}|
-
+| :--- | :--- | :--- | :--- | :--- |
+|success|boolean|True|Whether the action was successful|True|
+  
 Example output:
 
 ```
 {
-  "member": {
-    "type": "0-Owner",
-    "accessAll": true,
-    "externalId": "external_id_123456",
-    "resetPasswordEnrolled": true,
-    "object": "member",
-    "id": "44d88612-fea8-a8f3-6de8-2e1278abb02f",
-    "userId": "44d88612-fea8-a8f3-6de8-2e1278abb02f",
-    "name": "John Smith",
-    "email": "user@example.com",
-    "twoFactorEnabled": true,
-    "status": "0-Invited",
-    "collections": [
-      {
-        "id": "44d88612-fea8-a8f3-6de8-2e1278abb02f",
-        "readOnly": true
-      }
-    ]
-  }
+  "success": true
 }
 ```
 
-#### Create a Member
+#### List All Collections
 
-This action creates a new member object by inviting a user to the organization.
+This action is used to return a list of your organization's collections. Collection objects listed in this call do not 
+include information about their associated groups
 
 ##### Input
-
-|Name|Type|Default|Required|Description|Enum|Example|
-|----|----|-------|--------|-----------|----|-------|
-|accessAll|boolean|None|True|Determines if this member can access all collections within the organization, or only the associated collections. If set to 'true', this option overrides any collection assignments|None|True|
-|collections|[]collection|None|False|The associated collections that this member can access|None|[]|
-|email|string|None|True|The member's email address|None|user@example.com|
-|externalId|string|None|False|External identifier for reference or linking this member to another system, such as a user directory|None|external_id_123456|
-|type|string|None|True|Organization user type|['0-Owner', '1-Admin', '2-User', '3-Manager']|0-Owner|
-
-Example input:
-
-```
-{
-  "accessAll": true,
-  "collections": [],
-  "email": "user@example.com",
-  "externalId": "external_id_123456",
-  "type": "0-Owner"
-}
-```
+  
+*This action does not contain any inputs.*
 
 ##### Output
 
 |Name|Type|Required|Description|Example|
-|----|----|--------|-----------|-------|
-|member|member|False|An organization member|{}|
-
+| :--- | :--- | :--- | :--- | :--- |
+|collections|[]collectionObject|False|List of collections|[]|
+  
 Example output:
 
 ```
 {
-  "member": {
-    "type": "0-Owner",
-    "accessAll": true,
-    "externalId": "external_id_123456",
-    "resetPasswordEnrolled": true,
-    "object": "member",
-    "id": "44d88612-fea8-a8f3-6de8-2e1278abb02f",
-    "userId": "44d88612-fea8-a8f3-6de8-2e1278abb02f",
-    "name": "John Smith",
-    "email": "user@example.com",
-    "twoFactorEnabled": true,
-    "status": "0-Invited",
-    "collections": [
-      {
-        "id": "44d88612-fea8-a8f3-6de8-2e1278abb02f",
-        "readOnly": true
-      }
-    ]
-  }
+  "collections": []
```

Comment:
Could you please add examples here? 😄

Reply:
Updated aall to have an correct output now! 😄